src/components/Shifts/AddShiftBar.tsx

```diff
@@ -1,34 +1,25 @@
 import React from "react";
-import { styled, Theme } from '@mui/material/styles';
-import {Alert, AlertProps, Button, Card, CardContent, Stack, Toolbar, Typography} from "@mui/material";
-import EventRepeatIcon from '@mui/icons-material/EventRepeat';
+import {Box, Button, Stack, Typography} from "@mui/material";
 
 export interface AddShiftBarProps {
   onClickGetStarted: () => void;
+  onViewStaffing: () => void;
```

Comment:
For consistency it would be good if they both either had 'click' in their name or didn't